package.json

```diff
@@ -154,7 +154,7 @@
     "amphtml-validator": "^1.0.31",
     "babel-loader": "^8.1.0",
     "babel-plugin-lodash": "^3.3.4",
-    "backstopjs": "3.8.8",
+    "backstopjs": "^5.0.6",
```

Comment:
This is nice (with the exception of all of the reference images needing to be updated 😄 ) – can you elaborate on why this version change was necessary?

Reply:
`3.8.8` doesn't have `dockerCommandTemplate` option which is important for us.